src/components/learner-credit-management/assignment-modal/AssignmentModalSummary.jsx

```diff
@@ -30,6 +30,7 @@ const AssignmentModalSummaryContents = ({
   return <AssignmentModalSummaryEmptyState />;
 };
 
+// TODO: Pass course runs to take into account hte individual run content_price
 const AssignmentModalSummary = ({
```

Comment:
Did we want to address this TODO in this PR?

Reply:
(Happy to take this one on, btw!)

Reply:
This was addressed further upstream within `AssignmentModalContent`, by changing the source of `contentPrice` to `courseRun.contentPrice`


``` jsx
    const allocationMetadata = isAssignEmailAddressesInputValueValid({
      learnerEmails,
      remainingBalance: spendAvailable,
      contentPrice,
    });
    setAssignmentAllocationMetadata(allocationMetadata);
    .
    .
    .
return (<AssignmentModalSummary
            course={course}
            learnerEmails={learnerEmails}
            assignmentAllocationMetadata={assignmentAllocationMetadata}
          />
  )
```
The

Reply:
We still pass only the `course` into `AssignmentModalSummaryContents` and, subsequently, `AssignmentModalSummaryLearnerList`, though? 🤔 

Then we ultimately render:

```jsx
<span className="ml-auto">
  {course.formattedPrice}
</span>
```

[clarification] Isn't this `course.formattedPrice` different from the run-specific price?